packages/@lwc/node-reactions/src/shared/fields.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { defineProperty } from './language';
+
+/**
+ * In IE11, symbols are expensive.
+ * Due to the nature of the symbol polyfill. This method abstract the
+ * creation of symbols, so we can fallback to string when native symbols
+ * are not supported. Note that we can't use typeof since it will fail when transpiling.
+ */
+// TODO: remove this dirty hack
```

Comment:
@caridy Is there something special about using Symbols in unit tests?
A bunch of unit tests are failing because the Symbol used in the `setInternalField` is different from the one used in `getInternalField`.